src/transformers/models/t5/modeling_t5.py

```diff
@@ -255,6 +260,19 @@ def forward(self, hidden_states):
         return self.weight * hidden_states
 
 
+try:
+    from apex.normalization import FusedRMSNorm
+
+    T5LayerNorm = FusedRMSNorm  # noqa
+
+    print("XXX: using FusedRMSNorm")
+except ImportError:
+    print("XXX: using T5LayerNorm")
+except Exception:
+    print("XXX: using T5LayerNorm: unknown exception")
+    pass
```

Comment:
This needs to be cleaned up using a `logger.info`. I don't think we want the last except block, an unknown exception should be raised.

Reply:
done, 

but I'm not sure about removing the last catch-all exception What if `apex` has a problem loading - now the user can't run the model. 

I added it specifically so that the model remains working if the speed up addition fails for any reason.

I'd add:

```
except Exception:
    logger.error("apex discovered but it failed to load, falling back to T5LayerNorm")
    pass
```

Thoughts?

Reply:
Works for me!

Reply:
done, but changed to `logger.warning` since it's not an error per se and shouldn't alarm the user too much.